src/main/java/org/folio/marccat/resources/TenantAPI.java

```diff
@@ -25,29 +34,30 @@
   private TenantRefService tenantRefService;
 
   @PostMapping
-  public ResponseEntity<String> create(
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl,
-    @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
-    @RequestBody TenantAttributes attributes
-  ) throws SQLException, IOException {
+  public ResponseEntity<String> create(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_URL_TO) String okapiUrlTo,
+      @RequestBody TenantAttributes attributes) throws SQLException, IOException {
     addHeaders(tenant, okapiUrl, okapiUrlTo);
     tenantService.createTenant(tenant, okapiUrl);
-    if(!okapiUrl.isEmpty())
+    if (!okapiUrl.isEmpty()) {
       tenantRefService.loadData(attributes, okapiHeaders);
-    return new ResponseEntity("Success", CREATED);
+      return new ResponseEntity("Success", CREATED);
+    } else {
+      return new ResponseEntity("Error", HttpStatus.BAD_GATEWAY);
+    }
+
   }
 
-  public void addHeaders(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant, @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo) {
+  public void addHeaders(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_URL_TO) String okapiUrlTo) {
     okapiHeaders.put(Global.OKAPI_TENANT_HEADER_NAME, tenant);
     okapiHeaders.put(Global.OKAPI_URL, okapiUrl);
-    okapiHeaders.put(Global.OKAPI_TO_URL, okapiUrlTo);
+    okapiHeaders.put(Global.OKAPI_URL_TO, okapiUrlTo);
   }
 
   @DeleteMapping
-  public ResponseEntity<Void> delete(
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant
-  ) throws SQLException {
+  public ResponseEntity<Void> delete(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant)
+      throws SQLException {
```

Comment:
Remove throws SQLException

Reply:
The function deleteTenant is empty. I don't know if it is right. The test of delete should be done too. I think it is better to do it in another PR

Reply:
It was not implemented because it is a very dangerous operation according to Mirko. It can be done in another PR.